tests/system/aiplatform/test_dataset.py

```diff
@@ -227,186 +164,156 @@ def test_get_nonexistent_dataset(self):
         """Ensure attempting to retrieve a dataset that doesn't exist raises
         a Google API core 404 exception."""
 
-        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
-
         # AI Platform service returns 404
         with pytest.raises(exceptions.NotFound):
             aiplatform.ImageDataset(dataset_name="0")
 
-    @pytest.mark.usefixtures("create_text_dataset", "delete_new_dataset")
-    def test_get_new_dataset_and_import(self, dataset_gapic_client, shared_state):
+    def test_get_new_dataset_and_import(self, dataset_gapic_client):
         """Retrieve new, empty dataset and import a text dataset using import().
         Then verify data items were successfully imported."""
 
-        assert shared_state["dataset_name"]
-        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
+        try:
+            text_dataset = aiplatform.TextDataset.create(
+                display_name=f"temp_sdk_integration_test_create_text_dataset_{uuid.uuid4()}",
+            )
 
-        my_dataset = aiplatform.TextDataset(dataset_name=shared_state["dataset_name"])
+            my_dataset = aiplatform.TextDataset(dataset_name=text_dataset.name)
 
-        data_items_pre_import = dataset_gapic_client.list_data_items(
-            parent=my_dataset.resource_name
-        )
+            data_items_pre_import = dataset_gapic_client.list_data_items(
+                parent=my_dataset.resource_name
+            )
 
-        assert len(list(data_items_pre_import)) == 0
+            assert len(list(data_items_pre_import)) == 0
 
-        # Blocking call to import
-        my_dataset.import_data(
-            gcs_source=_TEST_TEXT_ENTITY_EXTRACTION_GCS_SOURCE,
-            import_schema_uri=_TEST_TEXT_ENTITY_IMPORT_SCHEMA,
-            import_request_timeout=None,
-        )
+            # Blocking call to import
+            my_dataset.import_data(
+                gcs_source=_TEST_TEXT_ENTITY_EXTRACTION_GCS_SOURCE,
+                import_schema_uri=_TEST_TEXT_ENTITY_IMPORT_SCHEMA,
+                import_request_timeout=600.0,
+            )
 
-        data_items_post_import = dataset_gapic_client.list_data_items(
-            parent=my_dataset.resource_name
-        )
+            data_items_post_import = dataset_gapic_client.list_data_items(
+                parent=my_dataset.resource_name
+            )
 
-        assert len(list(data_items_post_import)) == 469
+            assert len(list(data_items_post_import)) == 469
+        finally:
+            if text_dataset is not None:
```

Comment:
This condition doesn't seem necessary. Preference to remove it. If this line is present or not present, referencing `text_dataset` would throw.